packages/service/src/service.ts

```diff
@@ -55,8 +56,16 @@ app.use(passport.session());
 
 app.post("/login", passport.authenticate("local", { successRedirect: "/" }));
 
-app.post("/document", function (req, resp) {
-  resp.send({ user: req.user });
+app.post("/document", async function (req, resp) {
+  if (!handleAuthCheck(req, resp)) {
+    return;
+  }
+  const dbDoc = await createDocument({ doc: new Y.Doc(), user: req.user });
+  resp.send({
+    user: req.user,
+    document: dbDoc.document,
+    user_document: dbDoc.user_document,
```

Comment:
It's not necessary to expose the join table in this way. It might be useful to return the list of users that are connected to a document, but this single user_document for the current user doesn't provide much value.